packages/ui-react/src/components/LayoutGrid/LayoutGrid.tsx

```diff
@@ -9,14 +9,15 @@ type Props<Item> = {
   columnCount: number;
   data: Item[];
   renderCell: (item: Item, tabIndex: number) => JSX.Element;
+  getCellKey: (item: Item, rowIndex: number, columnIndex: number) => string;
```

Comment:
I think you're not using the `rowIndex` and `columnIndex`.
(But perhaps you can just use `item.mediaid`, see next comment)

Reply:
The `LayoutGrid` is made to be flexible (with any data). I initially added the `Item extends { mediaid: string }` but later added the `getCellKey` callback, but I forgot to remove the extends part.

I added `rowIndex` and `columnIndex` just in case, but it's not used in the current implementation.

Reply:
Got it. Then let's just remove the `extends`. :-)